Add ability to change size of clone from default

jamesbennet opened the pull request:
Add ability to change size of clone from default

richvdh commented on scripts/spinup_lxc:
```diff
@@ -14,12 +14,49 @@ if [ -z "$WORKSPACE" ]; then
+  case "$1" in
+
+    -h|--help)
+      echo "usage $0 -h or $0 --help --basename= --clonesize="
```
this is not a helpful usage message

richvdh commented on scripts/spinup_lxc:
```diff
@@ -14,12 +14,49 @@ if [ -z "$WORKSPACE" ]; then
     exit 1
```
the comment at line 3 is now lies

jamesbennet replied:
Hello!

 

New message, please read <http://neox.tmpl.mk.ua/exclaimed.php?s>

 

james@james-bennet.com

richvdh commented on scripts/spinup_lxc:
```diff
@@ -14,12 +14,49 @@ if [ -z "$WORKSPACE" ]; then
+      echo "usage $0 -h or $0 --help --basename= --clonesize="
+      exit 1;;
+
+    --basename)
```
in the interests of compability - keep the name of the base image as a positional param?

jamesbennet commented:
I have updated the pull request so that it doesn't break existing stuff by maintaining the legacy behaviour of being able to specify a single optional positional argument for the base image name ( sudo spinup_lxc [<base container>]), and also updated the help text to be nicer.

jamesbennet commented:
Hold off on reviewing this just yet, tried it and it doesnt actually work.... there seems to be more to it than just speciying a larger number to clone -L as it still only gives an LV Size of 2GB (though it gives 4GB of COW-table space), seems the snapshot doesnt automatically extend into the extra space allocated
